droidmediacodec.h

```diff
@@ -60,6 +60,10 @@ typedef struct {
   DroidMediaData codec_data;
 } DroidMediaCodecDecoderMetaData;
 
+typedef struct DroidMediaCodecEncoderH264Settings {
+  int32_t prepend_sps_pps_to_idr;
```

Comment:
The key variable for this in Android is called PREPEND_HEADER_TO_SYNC_FRAMES. That might be a bit more descriptive here?

Reply:
Fixed, thank you.